codegen_glibmm/templates/stub.h.templ

```diff
@@ -57,6 +57,9 @@ protected:
 
     void {{ signal.name }}_emitter({{ signal.args|map(attribute='cpptype_out')|join(', ') }});
     sigc::signal<void, {{ signal.args|map(attribute='cpptype_out')|join(', ') }}> {{ signal.name }}_signal;
+
+    void {{ signal.name }}_selectiveEmitter(std::vector<Glib::ustring>&, {{ signal.args|map(attribute='cpptype_out')|join(', ') }});
```

Comment:
What if we added an additional parameter to `emitter()`, instead (empty by default)?

Reply:
Do you mean "emitter(const std::vector<Glib::ustring> &destination_bus_names = {}, ...)" default parameter?
So do you expect "XXX_signal.emit(DATA)" using broadcast and "XXX_signal.emit(ClientList, DATA)" using multicast ?

But, it will be binding signal to XXX_signal(mem_fun(this, &emitter), then it is required 
clientList parameter again like this "XXX_signal.emit({}, DATA)".
Because binded XXX_signal.emit() will don't know emitter function's default parameter. 
 
therefore I patched dedicated signal variable for multicast.

Reply:
Yes I meant that, though I'd leave the list of destinations as second parameter, not as first one.

But now I see what the problem is. At the moment I cannot come up with any better solution, than what you already found, though I'm still not 100% conviced by it. Please let me wait for a few days, I wonder if we can come up with a better solution. @martin-ejdestig @erikboto @kursatkobya , any thoughts?